SIL.ReleaseTasks/SetReleaseNotesProperty.cs

```diff
@@ -190,12 +149,39 @@ private string ConvertLatestChangelog(int level, int skipUntilLevel = -1)
 							_currentIndex = _markdownLines.Length;
 						break;
 					default: 
-						if (currentLine.StartsWith("-") && _filterRegex.IsMatch(currentLine) && FilterEntries)
+						if (FilterEntries)
 						{
-							if (currentLine.StartsWith($"- [{PackageId}]"))
+							int m = _currentIndex;
+							if (_filterRegex.IsMatch(currentLine) && !String.IsNullOrEmpty(currentLine))
```

Comment:
```suggestion
							if (_filterRegex.IsMatch(currentLine))
```

We already checked for an empty line above the switch statement.